src/features.rs

```diff
@@ -81,11 +81,11 @@ macro_rules! si {
     ($($tt:tt)*) => {};
 }
 
-/// Expands the given block of code when `uom` is compiled with the `std` feature.
+/// Expands the given block of code when `uom` is compiled with either the `std` or the `libm` feature.
 #[doc(hidden)]
 #[macro_export]
-#[cfg(feature = "std")]
-macro_rules! std {
+#[cfg(any(feature = "std", feature = "libm"))]
+macro_rules! float {
```

Comment:
Revert these changes and instead add two new macros, `std_libm`, that can be used to conditionally expand the nested code when either of the features is enabled. See `autoconvert_test` as an example.

Reply:
I did so in the first place, however, I think this doesn't make sense (why I unified the two). There are only two places allover the `uom` src where the `std!` macro was used (both in `src/system.rs`). And in both places, the usage of `std!` is actually only about using using float related operations IIRC. So it makes sense to not use the `std!` macro at all, but always the `std_libm!` feature.

Based on that assumption, it also makes sense to remove the `std!` completely (since it is unused) and I think ~~`float!` is more accurate than `std_libm!`, but that's up to debate (most descriptive of cause would be `std_or_libm!`)~~ `float` creates name collisions, so I used `std_or_libm` instead . 

Is this a convincing argument?
Do you want to keep the `std!`?